desktop/app.js

```diff
@@ -162,7 +162,7 @@ function createMenuTemplate( settings ) {
 			type: 'separator'
 		} );
 		fileMenu[ 'submenu' ].push( {
-			label: 'Exit',
+			label: '&Exit',
```

Comment:
@dmsnell it's been a long while indeed. I thought I sent the suggested change long time ago, i forget to push apparently. Cheers,